lib/src/components/NxTextLink/NxTextLink.tsx

```diff
@@ -30,7 +31,8 @@ const NxTextLink = forwardRef<HTMLAnchorElement, Props>(
 
           className = classnames('nx-text-link', specifiedClassName, {
             'nx-text-link--external': external,
-            'nx-text-link--truncate': truncate
+            'nx-text-link--truncate': truncate,
+            'nx-text-link--disabled': disabled
```

Comment:
Why this class and not just `disabled`?

Reply:
I was following the pattern here but would like to know if there is any potential issue to use `'nx-text-link--disabled'` class? Thanks.

Reply:
> I was following the pattern here but would like to know if there is any potential issue to use 'nx-text-link--disabled' class? Thanks.

There's a handful of classes that represent common concepts applicable across a variety of components, which we actually don't follow BEM for.  `disabled` is one of them.  See the Utility Classes section on https://gallery.sonatype.dev/#/pages/Styling%20Components

Reply:
> There's a handful of classes that represent common concepts applicable across a variety of components, which we actually don't follow BEM for. disabled is one of them. See the Utility Classes section on https://gallery.sonatype.dev/#/pages/Styling%20Components

Thanks for the info, will update it accordingly. 

I noticed the following components do have `disabled` as a modifier classname, are they exceptional?
NxCollapsibleItems/ NxRadio/ NxCheckbox/ NxToggle

Reply:
> will update it accordingly.

a47cb4691f65a1ba52a029a9cf05e5e4e79dc9fb

Reply:
> I noticed the following components do have disabled as a modifier classname, are they exceptional?
NxCollapsibleItems/ NxRadio/ NxCheckbox/ NxToggle

Maybe we just aren't quite as consistent about this as we'd like to be